changes/30082-ms-office-version-fix

```diff
@@ -0,0 +1 @@
+* Microsoft office Fleet maintained apps for Mac are now added with short version to match software inventory
```

Comment:
```suggestion
```

Since this isn't actually tied to a server release, we shouldn't include a changes file here.